SmartPlaces.Facilities/lib/IngestionManager.Mapped/src/MappedGraphIngestionProcessor.cs

```
@@ -150,7 +156,13 @@ private async Task UpdateOutputSiteAsync(JsonElement siteElement, CancellationTo
                                    IDictionary<string, BasicRelationship> relationships,
                                    JsonElement targetElement)
         {
-            var basicDtId = targetElement.GetProperty("id").ToString();
+            if (!targetElement.TryGetProperty("id", out var basicDtIdProp))
+            {
```

Any reason for skipping the log statements on the things below a Site?

---

If there is no id on the Site element itself, something is wrong with the site. I haven't seen a missing site id... yet

---

If the buildingId is empty, there is something wrong with the data. BuildingId is required.

---

From this return, how would the end user know that their data is wrong without intimate knowledge on the required schema? This seems to be silently swallowing the bug.